### PR TITLE
Deflect: Transmit pinch & touch points events to clients [DISCL-386]

### PR DESCRIPTION
**Pull request by rdumusc**
Needs https://github.com/BlueBrain/Deflect/pull/128

### CODE REVIEW

File: tide/master/multitouch/DoubleTapDetector.cpp [hunk 1]
```diff
@@ -0,0 +1,104 @@
+/*********************************************************************/
```
Comment (tribal-tec) [inline tide/master/multitouch/DoubleTapDetector.cpp]: funny to see the gestureRecognizers return with a different name :)

File: tide/master/multitouch/MathUtils.h [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+#define MATHUTILS_H
+
+#include <QRectF>
+
+#include "types.h"
+
+/**
+ * Utility math functions used by multitouch gesture detectors.
+ */
+class MathUtils
```
Comment (tribal-tec) [inline tide/master/multitouch/MathUtils.h]: namespace instead of class & only-statics?

Comment (rdumusc) [inline tide/master/multitouch/MathUtils.h]: done

**Comment by tribal-tec** [comment]
+1

**Pull request merged**